include/boost/config/compiler/visualc.hpp

```diff
@@ -303,8 +303,10 @@
 #     define BOOST_COMPILER_VERSION 11.0
 #   elif _MSC_VER < 1900
 #     define BOOST_COMPILER_VERSION 12.0
-#   elif _MSC_VER < 2000
+#   elif _MSC_VER < 1910
 #     define BOOST_COMPILER_VERSION 14.0
+#   elif _MSC_VER < 1911
+#     define BOOST_COMPILER_VERSION 15.0
```

Comment:
This should be 14.10 in order to be consistend with the latest Boost.Build patches.
`#     define BOOST_COMPILER_VERSION 14.10`